### DISCUSSION TITLE
No status icon in VS Code 1.70.0

### DISCUSSION THREAD

**Discussion by DrWheelicus** [community/community#28392] [Copilot Conversations] [answered]
I have been using and **loving** Copilot for the past few months but today it seems to be testing my patience. There are no suggestions being shown and the Copilot logo is nowhere to be found on the status bar. I'm really hoping that someone knows how to fix this!

### Steps already taken to resolve this:

- Looked through the extensions to make sure Copilot was installed
- Reinstalled Copilot
- Checked View > Output for any errors 
    - Copilot doesn't even exist in the dropdown
- Checked the Runtime Status
    - ![image](https://user-images.githubusercontent.com/42521312/182938233-9968d539-1f46-4dbb-9f5a-b9efa53720a1.png)
- Looked at #12284 but no luck

**Comment by DrWheelicus** [comment]
### Update
Rolling back to the previous version seems to have fixed it but I would ideally like to figure out why

**Comment by davecheney** [comment, marked as answer]
@DrWheelicus thank you for your report. I'm running vscode 1.70 (just updated this morning) and copilot is working for me with the nightly[^1] version of the extension. Would you perhaps be able to try and see if this resolves your issue[^2]?

[^1]: https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly
[^2]: the nightly version is effectively the head of the branch that we release the main extension from, in effect you're just getting access to the next release faster.

**Reply by DrWheelicus** [reply]
Nightly was the answer! Thank you
